robotis-op2/robot-window.md

```diff
@@ -0,0 +1,18 @@
+# Robot Window
+
+When you double click on the robot, a new window appears. This window is called
+`Robot Window`, it has several tabs allowing you to perform different things.
```

Comment:
In this chapter, to me it would be ok to use once the version with the capital letter because it really describes what it is.
But in other chapters where the focus is on something else, using lower case make the text more readable.